chainguard-source

```diff
@@ -76,20 +77,45 @@ warn_about_resources() {
 		echo "And this will consume a lot of network bandwith."
 		echo "Your connection may be throttled by the remote servers."
 		echo
-		while true; do
-			echo -n "Do you want to continue? [y/n]: "
-			read -n1 answer
-			case "$answer" in
-				y|Y)
-					echo
-					break
-				;;
-				*)
-					echo
-					exit 1
-				;;
-			esac
-		done
+		echo -n "Do you want to continue? [y/N]: "
+		read -n1 answer
+		case "$answer" in
+			y|Y)
+				echo
+				break
```

Comment:
as the while loop has been removed this break seems superflous

Reply:
Whoops!